src/OrchardCore.Modules/OrchardCore.Queries/Deployment/QueryBasedContentDeploymentSource.cs

```diff
@@ -26,7 +26,7 @@ protected override async Task ProcessAsync(QueryBasedContentDeploymentStep step,
             return;
         }
 
-        if (!query.CanReturnContentItems || !query.ReturnContentItems)
+        if (!query.CanReturnContentItems && !query.ReturnContentItems)
```

Comment:
Do we know what is the difference between the CanReturnContentItems and ReturnContentItems?

Reply:
`CanReturnContentItems` seems to me like an abandoned view model property. It's not actually used anywhere, I've removed it.